src/main/java/net/zenoc/gallium/plugin/java/JavaPlugin.java

```diff
@@ -0,0 +1,17 @@
+package net.zenoc.gallium.plugin.java;
+
+import net.zenoc.gallium.plugin.Plugin;
+
+/**
+ * A Java plugin<br>
+ *
+ * Your Java plugin's main class should extend this<br>
+ * <pre>
+ *     public class TestPlugin extends JavaPlugin {
+ *         // Your code here
+ *     }
+ * </pre>
+ */
+public class JavaPlugin extends Plugin {
```

Comment:
this doesn't appear to add anything...

Reply:
In case at some point down the road other languages get support, but that probably won't happen